react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react'
+
+
+export default function ToggleButton( { text, color} ) {
+
+  const [isClicked, setIsClick] = useState(false)
+
+  function handleClick() {
+    setIsClick(!isClicked)
+    console.log(isClicked)
```

Comment:
I'd like you to log `isClicked` 3 times so you can see how/when it changes:
1. During rendering (at the top of the ToggleButton function)
2. Before you call `setIsClicked`
3. After you call `setIsClicked`